restli-server/src/main/java/com/linkedin/restli/internal/server/ServerResourceContext.java

```diff
@@ -46,6 +46,17 @@
  */
 public interface ServerResourceContext extends ResourceContext
 {
+  /**
+   * Local attribute key for request cookies. Value should be a {@link List} of {@link HttpCookie} objects.
+   */
+  public static final String CONTEXT_COOKIES_KEY = "$Cookies$";
```

Comment:
Not sure why these constants are put in `ServerResrouceContext` interface? As I see it, they are more related to request?

Reply:
Because they are used in the Server resource context's local attributes.